app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express")
+const app = express();
+
+// controller import
+const { topicsController } = require("./controllers/app.controller.js")
+
+// routing
+app.get("/api/topics", topicsController)
+
+
```

Comment:
where are you handling the error that you have tested for? You may have accidentally tested for a http error, which you didn't write and therefore shouldn't test for.